### PR TITLE
Optimize Town Hall Page

### PR DESCRIPTION
**Pull request by yeadamk**
## Overview

This is an attempt further modularize the town hall page for better usability.
**For later:** Replace the `` tags with `next/image` for image optimization

## Changes (in progress)

`pages/town-hall.js`
- Displays the text "RSVP Coming Soon!" when the link to RSVP is empty on Google Sheets
- Displays the text "Stay Tuned for the Survey!" when the link to Survey is empty on Google Sheets
- Added comment for clarify (regarding the `TOWN_HALL_2021_WINTER_VIDEO`"

`pages/town-hall/`
- Removed previous town hall pages

Other minor fixes...

### CODE REVIEW

File: components/Navbar.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ export default function Navbar() {
           <a id="nav-title" className="nav-section left" aria-label="acm home">
             {/* TODO: resolve next/image issue */}
             {/* eslint-disable-next-line @next/next/no-img-element */}
-            <img src={'/images/acm_wordmark_chapter.svg'} id="acm-logo" alt="ACM at UCLA" />
+            <img src={'/images/acm_wordmark_chapter.svg'} class="acm-logo" alt="ACM at UCLA" />
```
Comment (snigdha-kansal) [inline components/Navbar.js:21]: is this change needed?

Comment (yeadamk) [reply components/Navbar.js:21]: I changed it in case we switch `img` to `next/image` in the future. It doesn't change anything visually.

File: stylelint.config.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ module.exports = {
     plugins: [
       'stylelint-scss',
     ],
-    ignoreFiles: ['dist/*', 'node_modules/*'],
+    ignoreFiles: ['dist/*', 'node_modules/*', 'build/**/*'],
```
Comment (snigdha-kansal) [inline stylelint.config.js:9]: ❤️

**Review by snigdha-kansal — APPROVED** [review]
Awesome work, Yeadam! 💯🔥 
The code is so neat, thanks for noticing and fixing the spacing too :)

Cat Apple

File: components/Navbar.js (hunk 1 above)
Comment (snigdha-kansal) [reply components/Navbar.js:21]: sounds good
